src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -60,6 +63,7 @@
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
     public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
+    public static final String INVALID_DETAILS_DESC = " " + PREFIX_DETAILS + " 1xchococake";
```

Comment:
I'm not sure if this should be tested since there are technically no invalid details at this point. I actually removed it from my editcommandtest because of this. Maybe we can introduce this in future iterations? Or should we leave it be in anticipation of the restricted details input?

Reply:
Thanks for the catch, accidentally left that in - will remove it!

Reply:
No worries, I think we should note that we removed details and should both keep an eye out for new restrictions on details 😆